.gitignore

```diff
@@ -24,3 +24,4 @@ _testmain.go
 *.prof
 
 coverage.out
+.idea
```

Comment:
Please revert this change. You can add to global ignore file.

Reply:
fixed :)